README.md

```diff
@@ -352,6 +352,11 @@ class MyBatchConsumer < Deimos::Consumer
       # Do something 
     end
   end
+
+  def post_process(records)
+    # ActiveRecord objects that have been used to save to the database.
+    # They contain primary keys only if associations are involved.
```

Comment:
The wording is a bit awkward... maybe

```
This method gets passed the ActiveRecord objects that were saved to the database.
They contain primary keys only if associations were also saved via bulk_import_id.
```

Reply:
`have been used to` I am sorry I should not flex my grammar. I will use your words.

I wanted to give the intent that these are not the active records returned after save. These are the records we created and passing to save the record.

Reply:
But... aren't they the same thing?

Reply:
I misunderstood that rails adds IDs back to the objects for every save.

I tested in rails console and attribute ID is saved to the object only when`object.save!` is called. For other scenarios, `object.save` doesnot really change the object attribute ID.